Can't get emotion example to build

👋 Hello! Thanks for contributing. Please use the template that matches your intention  

_I am..._


| -------------------------------------------------------------------------------------------------
| Reporting a bug or issue
| -------------------------------------------------------------------------------------------------

**Expected behavior:**
running `yarn` or `npm install` to work on the `emotion` example

**Observed behavior:**
Long error but basically 

```
error Error while building ./my-command.js
./node_modules/react-sketchapp/lib/module/buildTree.js
Module not found: Error: Can't resolve 'react-test-renderer' in '/Users/localuser/code/github/react-sketchapp/examples/emotion/node_modules/react-sketchapp/lib/module'
```

**How to reproduce:**
running `yarn` or `npm install` on the `emotion` example

**Sketch version:**
70.6

**Please attach screenshots, a zip file of your project, and/or a link to your github project**
Use `examples/emotion`

--- a/examples/emotion/package.json
+++ b/examples/emotion/package.json
@@ -21,6 +21,7 @@
     "emotion-primitives": "^1.0.0-beta.6",
     "react": "^16.4.1",
     "react-primitives": "^0.6.0",
-    "react-sketchapp": "^3.0.0"
+    "react-sketchapp": "^3.0.0",
+    "react-test-renderer": "^16.3.2"
   }
 }

